yt/frontends/sph/fields.py

```diff
@@ -28,6 +28,8 @@ class SPHFieldInfo(FieldInfoContainer):
         ("InitialMass", ("code_mass", [], None)),
         ("TrueMass", ("code_mass", [], None)),
         ("ElevenMetalMasses", ("code_mass", [], None)),
+        ("ColdFraction", ("", ["cold_fraction"], None)),
```

Comment:
What are `ColdFraction` and `HotTemperature` fields?  I've never seen these referenced before.